ci/manual_release.md

```diff
@@ -1,3 +1,144 @@
+# Creating a new Knative release locally
+
+**Note:** This should not be used for official releases.
+
+This is intended for debugging and testing purposes.
+
+## The release.sh script
+
+The `release.sh` script automates the creation of Knative releases, either nightly
```

Comment:
would be nice with a hyperlink for the script

Reply:
This is per repo, so I can't link it here. Clarified.